Create crud skeleton (#3629)
shopsys/shopsys: docs/administration/crud-controller/getting-started/creating-a-new-crud-controller.md

# Creating a new Crud Controller

This guide will show you how to create a new Crud Controller in your project.

## 1. Create a new Crud Controller

> Create a new Crud Controller by extending the `Shopsys\AdministrationBundle\Controller\AbstractCrudController` class and must have defined attribute `#[CrudController]` with the entity class name.

Reviewer: ```suggestion
Create a new Crud Controller by extending the `Shopsys\AdministrationBundle\Controller\AbstractCrudController` class and on your class, define attribute `#[CrudController]` with the entity class name.
```

Author: fixed

protected function configure(CrudConfig $config): CrudConfig
{
    return $config

>         ->setTitle(PageType::LIST, t('My new Crud Controller')) // Set the title of the list page

Reviewer 2: it's actually `ActionType` in the latest implementation... and it seems weird to use actionType here, but I can live with that

Author: fixed

>         ->setTitle(PageType::LIST, t('My new Crud Controller')) // Set the title of the list page

Reviewer 2: Is the main idea here that we would be able to call it multiple times to set each page? 
like
```
->setTitle(ActionType::LIST, t('List products'))
->setTitle(ActionType::EDIT, t(Edit product'))
```

Author: yes exactly
